frontend/static/js/form.js

```diff
@@ -96,14 +96,27 @@ Form.prototype = {
   }
 };
 
-function statusString(status, domain) {
+function statusString(status, issues, domain) {
   switch (status.status) {
     case 'unknown':
       return 'Status: ' + domain + ' is not preloaded.';
     case 'pending':
       return 'Status: ' + domain + ' is pending submission to the preload list.';
     case 'preloaded':
-      return 'Status: ' + domain + ' is currently preloaded.'
+      if (status.bulk) {
+        return 'Status: ' + domain + ' is currently preloaded.';
+      } else {
+        switch (worstIssues(issues)) {
+          case 'none':
+            return 'Status: ' + domain + ' is currently preloaded.';
+          case 'warnings':
+          case 'errors':
+            return 'Status: ' + domain + ' is currently preloaded, but missing current requirements.';
```

Comment:
I would say:

    return 'Status: ' + domain + ' is currently preloaded, but no longer meets the requirements. It may be at risk of removal';